deploy/values/cue.mod/gen/k8s.io/api/core/v1/annotation_key_constants_go_gen.cue

```diff
@@ -0,0 +1,136 @@
+// Code generated by cue get go. DO NOT EDIT.
+
+//cue:generate cue get go k8s.io/api/core/v1
+
+package v1
```

Comment:
Can we add these files to a `.gitignore`? Cue uses Go under the hood to generate these types if I recall correctly, and I think all this can be generated on demand & is properly versioned